M2/Macaulay2/m2/hypertext.m2

```diff
@@ -346,6 +355,14 @@ addAttribute(TD,     htmlAttr | {"colspan", "headers", "rowspan"})
 addAttribute(TH,     htmlAttr | {"colspan", "headers", "rowspan"})
 addAttribute(IMG,    htmlAttr | {"alt", "src", "srcset", "width", "height",
 	"sizes", "crossorigin", "longdesc", "referrerpolicy", "ismap", "usemap"})
+addAttribute(OL, htmlAttr | {"start"=>"0", "reversed", "type"})
```

Comment:
@pzinn - What was the rationale behind giving `"start"` an initial value of 0?  This gives inconsistent results between `net` and `html`.  For example:

```m2
i1 : beginDocumentation()

i2 : OL{"foo", "bar", "baz"}

o2 =   1. foo
       2. bar
       3. baz

i3 : html oo

o3 = <ol start="0">
       <li>
     foo  </li>
       <li>
     bar  </li>
       <li>
     baz  </li>
     </ol>
```
And in my browser, this gives:

![image](https://user-images.githubusercontent.com/1992248/224520681-c2096dd6-849c-4800-aa51-f4884f844771.png)

Also, `make validate-html` now fails on any doc files with an `<ol>` tag, e.g.,:

```
validating: Macaulay2Doc/html/_writing_spdocumentation.html
 *** invalid HTML: /<<PKGBUILDDIR>>/M2/usr-dist/common/share/doc/Macaulay2/Macaulay2Doc/html/_writing_spdocumentation.html
error: line 72: there is no attribute "start"
```

Reply:
The rationale is that all counters start at 0 in Macaulay2, including `NumberedVerticalList`.
the `Net` output is a bug which I will fix. 
I don't understand the validate issue -- `OL` does have the start attribute...

Reply:
Ok, that makes sense.

`start` is an attribute for `<ol>` tags in HTML5, but not in XHTML 1.1, which is the spec that's used for validation.

Reply:
I don't understand why `validate` doesn't refer to the attributes that are defined in `hypertext.m2`...

Reply:
The issue isn't with `validate` but with `make validate-html`, which is a different thing.  It runs `util/validator-runner.py`, which runs the check from https://validator.w3.org/ locally.

Reply:
got it. this is outside my area of expertise then :-P

Reply:
No worries!  I'll see if I can figure something out.